cvat-canvas/src/typescript/editHandler.ts

```diff
@@ -380,6 +368,21 @@ export class EditHandlerImpl implements EditHandler {
     }
 
     private closeEditing(): void {
+        if (this.isEditing && this.editData.state.shapeType === 'polyline') {
+            const { offset } = this.geometry;
+            const head = this.editedShape.attr('points').split(' ').slice(0, this.editData.pointID).join(' ');
+            const stringifiedPoints = `${head} ${this.editLine.node.getAttribute('points').slice(0, -2)}`;
+            const points = pointsToNumberArray(stringifiedPoints)
+                .slice(0, -2)
+                .map((coord: number): number => coord - offset);
+            if (points.length >= 2 * 2) { // minimumPoints * 2
+                const { state } = this.editData;
+                this.onEditDone(state, points);
+            }
+        } else {
+            this.onEditDone(null, null);
+        }
+        this.isEditing = false;
```

Comment:
- I believe we should set `this.isEditing = false` in `release` method
- I believe we should set `this.isEditing = true` in `initEditing` method